include/staff/apikey.inc.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php`
```

Comment:
@rbukovansky 
Looks like there is a backtick too much here - you'd better remove it ;)